### DISCUSSION TITLE
Obtain SSL certificate for custom domain from Route 53.

### DISCUSSION THREAD

**Discussion by SAGITOR** [vercel/vercel#8040] [Help]
Hello!, I added my custom domain about 10 hours ago and it's still stuck at the "Generate SSL Certificate" step, I checked the settings as mentioned in the documentation and everything seems fine. I don't understand if there is a problem with my DNS manager or with the configuration itself, if someone could explain me in detail what to check I would appreciate it 😀. (I attach images of my configuration)

![vercel-settings](https://user-images.githubusercontent.com/62491167/176305448-aaa92100-a143-4ab8-ae80-853f2286f647.png)
![route53-settings](https://user-images.githubusercontent.com/62491167/176305454-f667f7fb-3a4d-443d-80d3-9df168e57914.png)

**Comment by amyegan** [comment]
Hi @SAGITOR. I think this may be a problem with your CAA record. I might just be misunderstanding the UI of your DNS manager, but it looks like you have globalsign.com and letsencrypt.org on the same record. Can you try splitting that into two separate records?
